.github/workflows/test.yml

```diff
@@ -18,6 +18,17 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: 3.8
+    - name: Check for cache
+      uses: actions/cache@v3
+      with:
+        key: linter-${{ runner.os }}-${{ hashFiles('setup.py') }}
+        path: |
+          ${{ env.pythonLocation }}
+          ${{ env.LD_LIBRARY_PATH }}
+          ${{ github.workspace }}/.pytest_cache
+          ${{ github.workspace }}/.mypy_cache
+          ${{ github.workspace }}/test/models/efficientnet
```

Comment:
Do you need `/test/models/efficientnet` and `~/.cache/torch/hub` in jobs that dont make use of it?